api/src/team/application/certification-center-invitation/certification-center-invitation.route.js

```diff
@@ -35,4 +35,27 @@ export const certificationCenterInvitationRoutes = [
       tags: ['api', 'certification-center', 'invitations'],
     },
   },
+  {
+    method: 'DELETE',
+    path: '/api/certification-center-invitations/{certificationCenterInvitationId}',
+    config: {
+      handler: (request, h) => certificationCenterInvitationController.cancelCertificationCenterInvitation(request, h),
+      pre: [
+        {
+          method: securityPreHandlers.checkUserIsAdminOfCertificationCenterWithCertificationCenterInvitationId,
```

Comment:
A faire si on doit _stubber_ le _prehandler_.

```suggestion
          method: (request, h) => securityPreHandlers.checkUserIsAdminOfCertificationCenterWithCertificationCenterInvitationId(request, h),
```